client/src/main/java/org/evosuite/coverage/aes/Spectrum.java

```diff
@@ -0,0 +1,1351 @@
+	package org.evosuite.coverage.aes;
```

Comment:
Is this class used anywhere? If so clean it up and indent the code, if it isn't used anywhere please remove it.